qsignature/src/org/qcmg/sig/Options.java

```diff
@@ -141,6 +146,7 @@
 		minMappingQuality = ((Integer) options.valueOf("minMappingQuality"));
 		minBaseQuality = ((Integer) options.valueOf("minBaseQuality"));
 		noOfThreads = ((Integer) options.valueOf("noOfThreads"));
+		maxCacheSize = ((Integer) options.valueOf("maxCacheSize"));
```

Comment:
"maxCacheSize" doesn't follow JP's docs rule, it also confuse by the word "size". How about something like  "--cached-vcfs", it means how many qsigniture VCF files can be stored in RAM.

Reply:
I provided both the maxCacheSize and max-cache-size options for this property as a bridge between the 2 worlds. I didn't want to just put in the new way as no other option for this tool follows that structure and so both were put in with the view of removing maxCacheSize when the other options are updated (out of scope for this PR)